### DISCUSSION TITLE
Thermal properties and pyrolysis data for apartment furniture

### DISCUSSION THREAD

**Discussion by MohamedS2022** [firemodels/fds#12821] [Q&A]
Hello FDS team,

Could you direct me to a trustworthy source for thermal properties and pyrolysis data concerning furniture material in apartments like beds, mattresses, refrigerators, washing machines, cookers, etc?

**Regards
Mohamed Sayed**

**Comment by mcgratta** [comment]
[UL Fire Safety Research Institute Materials and Products Database](https://materials.fsri.org/)

**Reply by MohamedS2022** [reply]
@mcgratta 
Many thanks for your great guidance, highly appreciated!
